### DISCUSSION TITLE
talk/sliding-in-style-south-coast-ma/

### DISCUSSION THREAD

**Discussion by giscus** [gadenbuie/garrickadenbuie-com#106] [Comments]
# talk/sliding-in-style-south-coast-ma/

Make stylish slides with {xaringanthemer} and a little bit of CSS.

https://www.garrickadenbuie.com/talk/sliding-in-style-south-coast-ma/

**Comment by simonedu** [comment]
Hi Garrick, 

Thank your fantastic packages! I have learned  a lot with your talks as well!

I was wondering if it were possible to add an image magnifier glass function to xaringanExtra. That would be awesome!

Cheers, 

Simón
